### PR TITLE
LPS-150127 [CP 2.0] Logic to support seats value equal -1

### PR DESCRIPTION
**Pull request by Lucas2691**
### **How it was developed?**
Change code to support infinity invites when support seat value comes equal "-1".

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/TeamMemberInputs/index.js [hunk 1]
```diff
@@ -48,7 +48,8 @@ const TeamMemberInputs = ({
 					disabled:
 						administratorsAssetsAvailable === 0 &&
 						isAdministratorOrRequestorRole &&
-						!isAdministratorOrRequestorRoleSelected,
+						!isAdministratorOrRequestorRoleSelected &&
+						administratorsAssetsAvailable !== -1,
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/TeamMemberInputs/index.js:52]:
```suggestion
						administratorsAssetsAvailable > 0,
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
Hi @Lucas2691, congrats for this PR. I made some slight suggestions, more about the condition :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js [hunk 2]
```diff
@@ -358,36 +363,39 @@ const InviteTeamMembersPage = ({
...
-									}   roles available: ${availableAdminsRoles} of ${
-										project.maxRequestors
-									}`}
-								</h5>
-
-								<p className="mb-0 text-neutral-7 text-paragraph-sm">
-									{`Only ${project.maxRequestors} member${
-										project.maxRequestors > 1 ? 's' : ''
-									} per project (including yourself) have
+						{project.maxRequestors !== -1 && (
```
[the remark this answers is not in the document]
Comment (Lucas2691) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js:366]: Done!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js [hunk 3]
```diff
@@ -262,7 +263,11 @@ const InviteTeamMembersPage = ({
 				name="invites"
 				render={({pop, push}) => (
 					<>
-						<div className="invites-form overflow-auto px-3">
+						<div
+							className={classNames('overflow-auto px-3', {
+								'invites-form': project.maxRequestors !== -1,
```
[the remark this answers is not in the document]
Comment (Lucas2691) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js:268]: Done!

**Review by joanagrego — APPROVED** [review]

**Review by jairmedeiros — APPROVED** [review]

**Review by igordmf — APPROVED** [review]
Good job!

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**